sim/aviron/src/main.zig

```diff
@@ -143,13 +155,17 @@ const Cli = struct {
     mcu: MCU = .atmega328p,
     info: bool = false,
     format: FileFormat = .elf,
+    break_pc: ?u24 = null,
```

Comment:
I would rename this to `breakpoint`, so `aviron --breakpoint=0x1234`.

Or `break-pc` for `aviron --break-pc=0x1234`